developersmeetup_2018/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: common
+---
+
+# The Second Annual JuMP Developers Workshop
+
+June 28-29, 2018, at the Institut de Mathématiques de Bordeaux, University of Bordeaux
+
+## Purpose
+
+This workshop is a follow-up to the [first annual JuMP Developers Workshop](http://www.juliaopt.org/developersmeetup/) held in June 2017 at MIT. The purpose of the workshop is to bring together developers of mathematical optimization software based around the JuMP "stack". These include developers writing solvers in Julia, interfacing solvers from Julia, writing JuMP extensions (for problem classes such as stochastic, robust, and multiobjective), and developing other optimization libraries that utilize JuMP.
+
+The workshop is a great opportunity to meet other developers working on and around JuMP, as well as to provide feedback to the core JuMP team concerning its future direction.
+
+## Location
+
+The workshop will be held at the Institut de Mathématiques de Bordeaux, University of Bordeaux on the Thursday and Friday of the week prior to ISMP 2018 (i.e. June 28-29).
+
+A variety of information about Bordeaux, as well as how to get there, and possible accommodation options, is available on the [ISMP conference website](https://ismp2018.sciencesconf.org).
+
+## Call for participants
+
+The workshop is open for anybody to attend, however it is aimed at JuMP developers rather than users. In particular, we encourage new contributors and those who have not met the core development team to attend.
```

Comment:
s/JuMP developers/developers of mathematical optimization software who use JuMP/